docs/deploy/docker.md

```diff
@@ -1 +1,64 @@
 # Docker
+
+Containers provide a lightweight, fast, and isolated infrastructure to run your applications.
+The application, dependencies, libraries, binaries, and configuration files are usually bundled into
+the container image and thus provide an easy solution to migrating your application anywhere.
+Containers are therefore flexible, consistent, and portable, and they can run on almost any
+architecture that provides a sufficient container infrastructure.
+For more information on docker please refer to https://docs.docker.com/get-started/overview/.
+
+This page will focus on using containers with AMD GPUs and ROCm software components on Linux.
+
+# Prequisites

```

Comment:
Please change to level 2 header, only the page title should be a level 1 header.